src/js/Content/Features/Store/App/FHideReportedTags.js

```diff
@@ -0,0 +1,43 @@
+import {HTMLParser} from "../../../../modulesCore";
+import {Feature} from "../../../modulesContent";
+import {Page} from "../../Page";
+
+export default class FHideReportedTags extends Feature {
+
+    checkPrerequisites() {
+        // This feature intends to remove reported tags mistakenly shown as user-defined, see #1423
+        this._yourTags = document.querySelectorAll(".app_tag.user_defined");
+        return this._yourTags.length > 0;
+    }
+
+    apply() {
+
+        /**
+         * The second argument passed to `InitAppTagModal()` is an array of user tags
+         * https://github.com/SteamDatabase/SteamTracking/blob/590b40f75e2ed3a1a314f64448d466a812b6a686/store.steampowered.com/public/javascript/app_tagging.js#L63
+         */
+        const userTags = HTMLParser.getVariableFromDom(/\[{"tagid".+}\],\s*(\[{"tagid".+}\])/);
```

Comment:
Need help with my feeble attempt at writing rexexp...

Reply:
I think it works fine here